src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserdaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
+    UserDao userDao = new UserdaoImpl();
+    Scanner scanner = new Scanner(System.in);
```

Comment:
let's use scanner as local variable — it's not describing the state of our class

Reply:
done